freshroastsr700_phidget/max31865.py

```diff
@@ -23,7 +24,6 @@
 
 import time, math
 import RPi.GPIO as GPIO
```

Comment:
This is problematic on other platforms, I tried this on a mac and I got this: 

pip install RPi
Collecting RPi
  Could not find a version that satisfies the requirement RPi (from versions: )
No matching distribution found for RPi
(roast10) MacBook-Pro-8:SR700_Artisan_Profile_Builder luca$ pip install rpi
Collecting rpi
  Could not find a version that satisfies the requirement rpi (from versions: )
No matching distribution found for rpi
(roast10) MacBook-Pro-8:SR700_Artisan_Profile_Builder luca$ pip install RPi.GPIO
Collecting RPi.GPIO
  Downloading https://files.pythonhosted.org/packages/af/2f/407b6e4cc8a0bdf434825a160bba1807991886b63cce16a5f1a6e1f24cdf/RPi.GPIO-0.6.5.tar.gz
Building wheels for collected packages: RPi.GPIO
  Building wheel for RPi.GPIO (setup.py) ... error
  Complete output from command /Users/luca/anaconda3/envs/roast10/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-install-71r07gor/RPi.GPIO/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" bdist_wheel -d /private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-wheel-3zi5nw_n --python-tag cp36:
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.7-x86_64-3.6
  creating build/lib.macosx-10.7-x86_64-3.6/RPi
  copying RPi/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/RPi
  creating build/lib.macosx-10.7-x86_64-3.6/RPi/GPIO
  copying RPi/GPIO/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/RPi/GPIO
  running build_ext
  building 'RPi._GPIO' extension
  creating build/temp.macosx-10.7-x86_64-3.6
  creating build/temp.macosx-10.7-x86_64-3.6/source
  gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/luca/anaconda3/envs/roast10/include -arch x86_64 -I/Users/luca/anaconda3/envs/roast10/include -arch x86_64 -I/Users/luca/anaconda3/envs/roast10/include/python3.6m -c source/py_gpio.c -o build/temp.macosx-10.7-x86_64-3.6/source/py_gpio.o
  source/py_gpio.c:87:4: error: function definition is not allowed here
     {
     ^
  source/py_gpio.c:143:10: warning: implicit declaration of function 'cleanup_one' is invalid in C99 [-Wimplicit-function-declaration]
           cleanup_one();
           ^
  source/py_gpio.c:143:10: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  source/py_gpio.c:200:24: error: function definition is not allowed here
     int setup_one(void) {
                         ^
  source/py_gpio.c:293:13: warning: implicit declaration of function 'setup_one' is invalid in C99 [-Wimplicit-function-declaration]
         if (!setup_one())
              ^
  source/py_gpio.c:293:13: warning: this function declaration is not a prototype [-Wstrict-prototypes]
  source/py_gpio.c:345:21: error: function definition is not allowed here
     int output(void) {
                      ^
  source/py_gpio.c:419:18: error: called object type 'PyObject *' (aka 'struct _object *') is not a function or function pointer
        if (!output())
             ~~~~~~^
  source/py_gpio.c:475:18: error: called object type 'PyObject *' (aka 'struct _object *') is not a function or function pointer
        if (!output())
             ~~~~~~^
  4 warnings and 5 errors generated.
  error: command 'gcc' failed with exit status 1
  
  ----------------------------------------
  Failed building wheel for RPi.GPIO
  Running setup.py clean for RPi.GPIO
Failed to build RPi.GPIO
Installing collected packages: RPi.GPIO
  Running setup.py install for RPi.GPIO ... error
    Complete output from command /Users/luca/anaconda3/envs/roast10/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-install-71r07gor/RPi.GPIO/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-record-vvtjv6dm/install-record.txt --single-version-externally-managed --compile:
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.7-x86_64-3.6
    creating build/lib.macosx-10.7-x86_64-3.6/RPi
    copying RPi/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/RPi
    creating build/lib.macosx-10.7-x86_64-3.6/RPi/GPIO
    copying RPi/GPIO/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/RPi/GPIO
    running build_ext
    building 'RPi._GPIO' extension
    creating build/temp.macosx-10.7-x86_64-3.6
    creating build/temp.macosx-10.7-x86_64-3.6/source
    gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/Users/luca/anaconda3/envs/roast10/include -arch x86_64 -I/Users/luca/anaconda3/envs/roast10/include -arch x86_64 -I/Users/luca/anaconda3/envs/roast10/include/python3.6m -c source/py_gpio.c -o build/temp.macosx-10.7-x86_64-3.6/source/py_gpio.o
    source/py_gpio.c:87:4: error: function definition is not allowed here
       {
       ^
    source/py_gpio.c:143:10: warning: implicit declaration of function 'cleanup_one' is invalid in C99 [-Wimplicit-function-declaration]
             cleanup_one();
             ^
    source/py_gpio.c:143:10: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    source/py_gpio.c:200:24: error: function definition is not allowed here
       int setup_one(void) {
                           ^
    source/py_gpio.c:293:13: warning: implicit declaration of function 'setup_one' is invalid in C99 [-Wimplicit-function-declaration]
           if (!setup_one())
                ^
    source/py_gpio.c:293:13: warning: this function declaration is not a prototype [-Wstrict-prototypes]
    source/py_gpio.c:345:21: error: function definition is not allowed here
       int output(void) {
                        ^
    source/py_gpio.c:419:18: error: called object type 'PyObject *' (aka 'struct _object *') is not a function or function pointer
          if (!output())
               ~~~~~~^
    source/py_gpio.c:475:18: error: called object type 'PyObject *' (aka 'struct _object *') is not a function or function pointer
          if (!output())
               ~~~~~~^
    4 warnings and 5 errors generated.
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/Users/luca/anaconda3/envs/roast10/bin/python -u -c "import setuptools, tokenize;__file__='/private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-install-71r07gor/RPi.GPIO/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-record-vvtjv6dm/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /private/var/folders/_x/4h571cc553v0lng0t870jll40000gn/T/pip-install-71r07gor/RPi.GPIO/